.github/workflows/run_teaspoons_e2e_cli_tests.yaml

```diff
@@ -0,0 +1,181 @@
+      
+      - name: Checkout cli repo
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ inputs.cli-version-ref }}
+          repository: DataBiosphere/terra-scientific-pipelines-service-cli
+          path: terra-scientific-pipelines-service-cli/
+      
+      - name: Install Poetry
+        uses: snok/install-poetry@v1
+
+      - name: Generate OAuth2 2.0 access token for Terra user
+        id: obtain_user_token
+        uses: google-github-actions/auth@v1
+        with:
+          token_format: 'access_token'
+          workload_identity_provider: 'projects/1038484894585/locations/global/workloadIdentityPools/github-wi-pool/providers/github-wi-provider'
+          service_account: 'firecloud-qa@broad-dsde-qa.iam.gserviceaccount.com'
+          access_token_scopes: 'profile, email, openid'
+          access_token_subject: 'hermione.owner@quality.firecloud.org'
+          export_environment_variables: false
+      
+      - name: Generate OAuth2 2.0 access token for admin user and billing project owner
+        id: obtain_admin_user_token
+        uses: google-github-actions/auth@v1
+        with:
+          token_format: 'access_token'
+          workload_identity_provider: 'projects/1038484894585/locations/global/workloadIdentityPools/github-wi-pool/providers/github-wi-provider'
+          service_account: 'firecloud-qa@broad-dsde-qa.iam.gserviceaccount.com'
+          access_token_scopes: 'openid, https://www.googleapis.com/auth/userinfo.email, https://www.googleapis.com/auth/userinfo.profile, https://www.googleapis.com/auth/cloud-billing'
+          access_token_subject: 'dumbledore.admin@quality.firecloud.org'
+          export_environment_variables: false
+      
+      - name: Setup Teaspoons resources
+        id: setup-resources
+        env:
+          BEE_NAME: ${{ inputs.bee-name }}
+          BILLING_PROJECT_NAME: ${{ inputs.billing-project-name }}
+          BILLING_ACCOUNT_NAME: 'billingAccounts/00708C-45D19D-27AAFA'
+          ADMIN_TOKEN: ${{ steps.obtain_admin_user_token.outputs.access_token }}
+          WDL_METHOD_VERSION: ${{ inputs.wdl-method-version }}
+          WORKSPACE_NAME: 'teaspoons_e2e_cli_test_workspace'
+        run: |
+          cd dsp-reusable-workflows/
+          python e2e-test/teaspoons_gcp_cli_e2e_test_setup.py
+        
+      - name: Run e2e test with CLI
+        id: run-cli-e2e-test
+        env: 
+          BEE_NAME: ${{ inputs.bee-name }}
+          USER_TOKEN: ${{ steps.obtain_user_token.outputs.access_token }}
+          MAX_WAIT_TIME: 1800  # 30 min in seconds
+          CHECK_INTERVAL: 120  # 2 minutes in seconds
+          INITIAL_WAIT: 600    # 10 minutes in seconds
+        run: |
+          set -eu
+          
+          cd terra-scientific-pipelines-service-cli/
+          poetry install
+
+          echo "### CONFIGURING CLI ###"
+          CONFIG_FILE=$(poetry run python -c "import terralab; print(terralab.__file__.replace('__init__.py','.terralab-cli-config'))")
```

Comment:
is this based on the name of the file or where does `.terralab-cli-config'` come from

Reply:
yeah that's just what the config file is named in the cli repo: https://github.com/DataBiosphere/terra-scientific-pipelines-service-cli/blob/main/terralab/.terralab-cli-config